### PR TITLE
Consolidate Ohio AGI variable

### LINKED ISSUE #3095: Consolidate Ohio AGI variable
need to create additions and subtractions list parameters and python files

### PR DESCRIPTION
**Pull request by WeihaoZ724**
Fixes #3095

### Summary
:sparkles::memo::card_file_box:

This pull request adds Ohio AGI additions and deductions parameters and variables to the policy engine model. It also updates the changelog entry to reflect the minor version bump and the new features.

> _`Ohio AGI` added_
> _New parameters and variables_
> _Autumn tax season_

### Walkthrough
*  Bump minor version and update changelog entry ([link](https://github.com/PolicyEngine/policyengine-us/pull/3111/files?diff=unified&w=0#diff-5885042762833a4aef51405a4036067e4e6aefa7a5091a2987d1d9d1fc2647e8L1-R3))
*  Add Ohio AGI additions and deductions parameters to the model ([link](https://github.com/PolicyEngine/policyengine-us/pull/3111/files?diff=unified&w=0#diff-bf0c690c31f5bcaa7a461836251e04f25db3987a3d05a5079cc6b673885cc974R1-R17), [link](https://github.com/PolicyEngine/policyengine-us/pull/3111/files?diff=unified&w=0#diff-d059c027d0387d2f106acdf46f94d8e6a2b0bfe51270bb8f3cc5885b5a63cdf0L1-R15))
*  Add Ohio AGI additions and deductions variables to the model ([link](https://github.com/PolicyEngine/policyengine-us/pull/3111/files?diff=unified&w=0#diff-d822b76124e7bef4aca77e01096d6e90a2321e98e6e4cbf7c5427b95ff829e56L1-R15), [link](https://github.com/PolicyEngine/policyengine-us/pull/3111/files?diff=unified&w=0#diff-d606df8bee643abf49ccaf4f24b9d0c14f25ab11d61b7f838231aad7fb539dc7L1-R15))

### CODE REVIEW

File: policyengine_us/parameters/gov/states/oh/tax/income/additions.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+description: Ohio counts these sources as additions.
+values:
+  2021-01-01:
+    - adjusted_gross_income",
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/oh/tax/income/additions.yaml:4]:
```suggestion
```

File: policyengine_us/variables/gov/states/oh/tax/income/adjusted_gross_income/oh_agi.py [hunk 2]
```diff
@@ -13,21 +13,6 @@ class oh_agi(Variable):
     )
     defined_for = StateCode.OH
 
-    adds = [
-        "adjusted_gross_income",
-        "oh_bonus_depreciation_add_back",
-        "oh_other_add_backs",
-    ]
+    adds = ["oh_additions"]
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/oh/tax/income/adjusted_gross_income/oh_agi.py:16]:
```suggestion
    adds = ["adjusted_gross_income", "oh_additions"]
```

File: changelog_entry.yaml
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]: need a git pull upstream master

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/oh/tax/income/deductions.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+description: Ohio counts these sources as deductions.
```
[the remark this answers is not in the document]
Comment (MaxGhenis) [reply policyengine_us/parameters/gov/states/oh/tax/income/deductions.yaml]: You resolved this without answering my question, and applied the wrong change. Per the tax form:
>Additions and deductions are applied to FAGI in order to arrive at Ohio adjusted gross income (OAGI).

File: changelog_entry.yaml
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]: please take a look at the tax form in Ohio and check whether the additions and subtractions are complete

**Review by PavelMakarchuk — APPROVED** [review]

File: changelog_entry.yaml
Comment (MaxGhenis) [reply changelog_entry.yaml:1]: Approving but please resolve this

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**